ts/src/index.ts

```diff
@@ -73,13 +73,28 @@ export async function initConfig(config: Config): Promise<ProfilerConfig> {
   let envSetConfig: Config = {};
   const val = process.env.GCLOUD_PROFILER_CONFIG;
   if (val) {
+    const envSetConfigBuf = fs.readFileSync(val);
```

Comment:
This will throw an exception if val points to a non-existing / non-readable file. Should this be handled?